server/src/main/java/org/apache/iotdb/db/query/executor/AggregationExecutor.java

```diff
@@ -267,22 +271,17 @@ private void aggregateWithValueFilter(List<AggregateResult> aggregateResults,
       TimeGenerator timestampGenerator, List<IReaderByTimestamp> readersOfSelectedSeries)
       throws IOException {
 
-    while (timestampGenerator.hasNext()) {
-
-      // generate timestamps for aggregate
-      long[] timeArray = new long[aggregateFetchSize];
-      int timeArrayLength = 0;
-      for (int cnt = 0; cnt < aggregateFetchSize; cnt++) {
```

Comment:
Since `aggregateFetchSize` is not used, it could be deleted from this class.

Reply:
done